src/main/resources/transactions.avro

```diff
@@ -0,0 +1,46 @@
+{
+        "namespace": "datagen.example",
+        "name": "transactions",
+        "type": "record",
+        "fields": [
+                {"name": "transaction_id", "type": {
+                    "type": "long",
+                    "arg.properties": {
+                        "iteration": {
+                            "start": 1
+                        }
+                    }
+                }},
+                {"name": "card_id", "type": {
+                    "type": "long",
+                    "arg.properties": {
+                        "range": {
+                            "min": 1,
+                            "max": 24
+                        }
+                    }
+                }},
+                {"name": "user_id", "type": {
+                    "type": "string",
+                    "arg.properties": {
+                        "regex": "User_[1-9]{0,1}"
```

Comment:
@bbejeck - do we want a broader user range for high partition counts keyed on userId? can we have a broader user range?

Reply:
This is consistent with the other schemas:

```
pageviews_schema.avro:                        "regex": "User_[1-9]{0,1}"
stock_trades_schema.avro:            "regex": "User_[1-9]{0,1}"
transactions.avro:                        "regex": "User_[1-9]{0,1}"
users_array_map_schema.avro:                        "regex": "User_[1-9]{0,1}"
users_schema.avro:                        "regex": "User_[1-9]{0,1}"
```

Reply:
Yup, fine with me. I just know that this was one area that Bill and I had been talking about. Something to consider for the future, if the need arises.